src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/projectcreation/model/ProjectCreationModel.kt

```diff
@@ -21,50 +22,81 @@ class ProjectCreationModel {
     )
     var sourceLanguageProperty: Language by property()
     var targetLanguageProperty: Language by property()
```

Comment:
As far as naming convention goes, I think these should be `var sourceLanguage: Language by property()`. These variables are delegating getters and setters to an underlying property, but are not properties themselves. The properties would be `val sourceLanguageProperty = getProperty(ProjectCreationMode::sourceLanguage)`